### ISSUE TITLE
Alt+D shortcut to highlight address bar not working with Windows 10 Ribbon

### ISSUE THREAD

**Issue by lcmbo** [valinet/ExplorerPatcher#2847]
The Alt+D shortcut to highlight address bar is not working with Windows 10 Ribbon

With File Explorer > Control Interface: Windows 10 Ribbon, alt+D does nothing.
With either "Windows 11 Command Bar", Alt+D behaves as normal

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

**Comment by Amrsatrio** [comment]
Can reproduce. 22621.3155. Workaround for now is to use F4.

**Comment by microka** [comment]
Me too. 23H2 22631.3155

**Comment by oyvindbjorlow** [comment]
I also got this bug after upgrading a few days ago. Can confirm F4 still works as a workaround.

**Comment by kumanobori** [comment]
I have same problem
Window 11 23H2, build 22631.3155
ExplorerPatcher 22621.2861.62.2
on default setting of ExplorerPatcher, this doesn't happen,

> Please include what settints you changed

diff of exported .reg file is also ok?

```
$ diff ExplorerPatcher_22621.2861.62.2_default.reg ExplorerPatcher_22621.2861.62.2.reg
12c12
< "SearchboxTaskbarMode"=dword:00000001
---
> "SearchboxTaskbarMode"=dword:00000000
28c28
< "TaskbarSmallIcons"=dword:00000000
---
> "TaskbarSmallIcons"=dword:00000001
38c38
< "ShowSecondsInSystemClock"=dword:00000000
---
> "ShowSecondsInSystemClock"=dword:00000001
42c42
< "TaskbarSD"=dword:00000001
---
> "TaskbarSD"=dword:00000000
54c54
< "IMEStyle"=dword:00000000
---
> "IMEStyle"=dword:00000001
57c57
< [-HKEY_CURRENT_USER\Software\Classes\CLSID\{86ca1aa0-34aa-4e8b-a509-50c905bae2a2}\InprocServer32]
---
> [HKEY_CURRENT_USER\Software\Classes\CLSID\{86ca1aa0-34aa-4e8b-a509-50c905bae2a2}\InprocServer32]
64c64
< ;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_FileExplorerCommandUI"=dword:00000000
---
> ;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_FileExplorerCommandUI"=dword:00000001
70c70
< ;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_DisableModernSearchBar"=dword:00000000
---
> ;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_DisableModernSearchBar"=dword:00000001
72c72
< "ShrinkExplorerAddressBar"=dword:00000000
---
> "ShrinkExplorerAddressBar"=dword:00000001
74c74
< "HideExplorerSearchBar"=dword:00000000
---
> "HideExplorerSearchBar"=dword:00000001
78c78
< "Start_ShowClassicMode"=dword:00000000
---
> "Start_ShowClassicMode"=dword:00000001
80c80
< "TaskbarAl"=dword:00000001
---
> "TaskbarAl"=dword:00000000
86,88c86,92
< ;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_StartDocked_DisableRecommendedSection"=dword:00000000
< [HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\StartPage]
< "MakeAllAppsDefault"=dword:00000000
---
> ;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_StartUI_ShowMoreTiles"=dword:00000000
> [HKEY_CURRENT_USER\Software\ExplorerPatcher]
> ;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_StartUI_EnableRoundedCorners"=dword:00000000
> [HKEY_CURRENT_USER\Software\ExplorerPatcher]
> ;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_ForceStartSize"=dword:00000000
> [HKEY_CURRENT_USER\Software\ExplorerPatcher]
> ;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_NoStartMenuMorePrograms"=dword:00000000
90c94
< "AltTabSettings"=dword:00000000
---
> "AltTabSettings"=dword:00000001

```

**Comment by kumanobori** [comment]
```
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_FileExplorerCommandUI"=dword:00000001
```

when I changed this item of my setting to default value (0), alt+d worked again.

---

![image](https://github.com/valinet/ExplorerPatcher/assets/43110758/b2792527-d781-4c1b-8ddb-2db143fa4809)
I have restored default setting and changed only item shown on picture, CONTROL INTERFACE.
As shown on picture, 
when I have selected "Windows 11",  alt+d works, 
when I have selected "Windows 10" or "Windows 7", alt+d doesn't work.

**Comment by pyrates999** [comment]
I said to post screen shots, not do a diff of a reg file.  But you've appeared to find the setting where it happens.

Please post the rest of your settings by doing screen shots.

**Comment by Amrsatrio** [comment]
No need. I can reproduce it on my main machine.

**Comment by gpz1100** [comment]
@Amrsatrio What is the full registry path of where you're changing this?

I have win11 pro 23h2 installed with the latest (as of 20240605) explorerpatch and am running into this issue as well.  I use alt-d quite often.  Thanks!

**Comment by onionmad** [comment]
I also experience this issue. Very annoying as I use alt+d in my daily work otherwise. Without changing settings, sometimes the shortcut works, but 90% of the time it does not work. 

![image](https://github.com/user-attachments/assets/c1816009-ac39-4044-be56-88af769f5416)


![image](https://github.com/user-attachments/assets/5ce17691-138b-4117-84cc-3d5a9310055f)

**Comment by pyrates999** [comment]
your OS build is from June 11/2024: https://support.microsoft.com/en-gb/topic/june-11-2024-kb5039212-os-builds-22621-3737-and-22631-3737-d7f574c0-2b13-48ca-a9fc-a63093b1a2c2

please install the latest OS updates.

You are also running an older release of EP: https://github.com/valinet/ExplorerPatcher/releases

Please read release notes on how you can exclude certain directories and files for windows defender and your AV if you run a third party AV.

**Comment by gpz1100** [comment]
Has there been any progress on making this work out of the box?  Changing the reg setting in the .reg file mentioned above does not seems to work on 23h2  - 22631.4460.  Using EP 22621.4317.67.1.

**Comment by 98gmarquee** [comment]
I didn't realize how much I used it... here I am, 15 minutes after installing EP and an hour after fiddling with W11 the first time and I'm finding how often I use alt+D on explorer. lol!

Edition	Windows 11 Pro
Version	24H2
Installed on	‎1/‎9/‎2025
OS build	26100.2605
Serial number	MP2GYA0W
Experience	Windows Feature Experience Pack 1000.26100.36.0

ExplorerPatcher
Version 22621.4317.67.1

![image](https://github.com/user-attachments/assets/14ce9699-cb74-4611-b855-0bff987b09dc)

**Comment by rctgamer3** [comment]
This is still an issue with Windows 11 25H2 26200.6901 / ExplorerPatcher 26100.4946.69.5 with either W7 bar or W10 ribbon style.

**Comment by Amrsatrio** [comment]
Yes, sorry about that. I still haven't dug up ExplorerFrame.dll to find the culprit. 🙏

**Comment by Amrsatrio** [comment]
Well guess what I made a fix for it, now working on 26100.6899. 69.6 soon. Thank you for the patience.

**Comment by Amrsatrio** [comment]
Fixed in 69.6.
